### DISCUSSION TITLE
Mindmap with multiple roots

### DISCUSSION THREAD

**Discussion by oktomus** [mermaid-js/mermaid#5267] [Q&A]
Looking at [Other Examples](https://mermaid.js.org/syntax/examples.html#larger-flowchart-with-some-styling), we can see that it's possible to create a mermaid code with multiple graphs inside that are not connected together: 

```
graph TB
    sq[Square shape] --> ci((Circle shape))

    subgraph A
        od>Odd shape]-- Two line<br/>edge comment --> ro
        di{Diamond with <br/> line break} -.-> ro(Rounded<br>square<br>shape)
        di==>ro2(Rounded square shape)
    end

    %% Notice that no text in shape are added here instead that is appended further down
    e --> od3>Really long text with linebreak<br>in an Odd shape]

    %% Comments after double percent signs
    e((Inner / circle<br>and some odd <br>special characters)) --> f(,.?!+-*ز)

    cyr[Cyrillic]-->cyr2((Circle shape Начало));

     classDef green fill:#9f6,stroke:#333,stroke-width:2px;
     classDef orange fill:#f96,stroke:#333,stroke-width:4px;
     class sq,e green
     class di orange
```

But with [mindmaps](https://mermaid.js.org/syntax/mindmap.html), it's not possible. The code bellow generates the error message `There can be only one root`.

```
mindmap
  root((Part A))
    Origins
      Long history
      ::icon(fa fa-book)
      Popularisation
        British popular psychology author Tony Buzan
    Research
      On effectiveness<br/>and features
      On Automatic creation
        Uses
            Creative techniques
            Strategic planning
            Argument mapping
  root((Part B)
    Tools
      Pen and paper
      Mermaid
```

Is it possible to have multiple graphs in a mindmap? I would like to use mindmaps instead of graph to have a circular connection instead of a hierarchical vertical or horizontal one.

**Comment by Piratepie** [comment]
I'm pretty sure [subgraphs](https://mermaid.js.org/syntax/flowchart.html#subgraphs) are specific to flowcharts as that's the only place they're documented.
